appveyor.yml

```diff
@@ -70,30 +70,22 @@ for:
 -
   matrix:
     only:
-    - APPVEYOR_BUILD_WORKER_IMAGE: *windows_image
+      - APPVEYOR_BUILD_WORKER_IMAGE: *windows_image
   install:
     - "SET PATH=%PYTHON%;%PYTHON%/Scripts;%PATH%"
     - "python --version"
-    - "IF %PYTHON:~-4% == -x64 (SET ARCH=x64) ELSE (SET ARCH=i386)"
-    - "IF %PYTHON:~-4% == -x64 (SET PLATFORM_WHEEL=win_amd64) ELSE (SET PLATFORM_WHEEL=win32)"
-    - ps: "Start-FileDownload https://mediaarea.net/download/binary/mediainfo/${Env:MEDIAINFO_VERSION}/MediaInfo_CLI_${Env:MEDIAINFO_VERSION}_Windows_${Env:ARCH}.zip"
-    - ps: "unzip -o MediaInfo_CLI_${Env:MEDIAINFO_VERSION}_Windows_${Env:ARCH}.zip LIBCURL.DLL"
-    - ps: "Start-FileDownload https://mediaarea.net/download/binary/libmediainfo0/${Env:MEDIAINFO_VERSION}/MediaInfo_DLL_${Env:MEDIAINFO_VERSION}_Windows_${Env:ARCH}_WithoutInstaller.7z"
-    - ps: "7z -y x MediaInfo_DLL_${Env:MEDIAINFO_VERSION}_Windows_${Env:ARCH}_WithoutInstaller.7z MediaInfo.dll Developers/License.html"
-    # Required for tests to pass with tox, Windows looks for DLLs in PATH
-    - ps: "Copy-Item -Path MediaInfo.dll -Destination ${Env:PYTHON}"
-    - "move MediaInfo.dll pymediainfo"
-    - "move Developers\\License.html docs"
-    - "pip install tox"
-  build: off
+    - "pip install pdm tox"
+    - "pdm install --no-self"
```

Comment:
Could you tell me what that does and why this is required in addition to pip install pdm?

Reply:
yes!